docs/quickstarts/python-quickstart.md

```diff
@@ -0,0 +1,203 @@
+---
+layout: docs
+title: Python Quickstart
+---
+
+<h1 class="page-header">Python Quickstart</h1>
+
+<p class="lead">This guide gets you started with gRPC in Python with a simple working example.</p>
+
+<div id="toc"></div>
+
+## Before you begin
+
+### Prerequisites
+
+[versions of pip, any other tools, etc. go here]
+
+If you haven't done so already, install the gRPC runtime and gRPC tools.
+
+### Install gRPC
+
+Install gRPC:
+
+```sh
+  $ pip install grpcio
+```
+
+Or, to install it system wide:
+
+```sh
+  $ sudo pip install grpcio
+```
+
+If you're on Windows, make sure you installed the `pip.exe` component when you
+installed Python. Invoke as above but with `pip.exe` instead of `pip` (you may
+also need to invoke from a `cmd.exe` ran as administrator):
+
+```sh
+  $ pip.exe install grpcio
```

Comment:
No, let's go forward without `virtualenv`. If a user doesn't yet know about `virtualenv` let's not take on the task of teaching them here.